parallel-install/pkg/downloader/downloader.go

```diff
@@ -0,0 +1,22 @@
+package downloader
+
+import (
+	"github.com/kyma-incubator/hydroform/parallel-install/pkg/git"
+)
+
+const kymaURL = "https://github.com/kyma-project/kyma"
```

Comment:
what about downloading sources from other repos?

Reply:
You can do that calling `git.CloneRevision()`, allows to download any repo on any revision/branch/PR and it is public so clients can use it.

This package is just a convenience for the Kyma repo but could actually be removed.

Reply:
what about handling zips?

Reply:
That is not supported at the moment, only git repos.
If we need zips we can add a curl like utility and a decompress utility.